website/addons/box/model.py

```diff
@@ -81,9 +81,7 @@ class BoxNodeSettings(StorageAddonBase, AddonOAuthNodeSettingsBase):
     oauth_provider = Box
     serializer = BoxSerializer
 
-    foreign_user_settings = fields.ForeignField(
-        'boxusersettings', backref='authorized'
-    )
+    foreign_user_settings = fields.ForeignField('boxusersettings')
```

Comment:
Actually, please remove this field entirely. It was last relevant during a migration that changed how `BoxNodeSettings` and `BoxUserSettings` objects are associated with each other. There was supposed to be a clean-up PR for this and other addons, but that may have been lost in a flood of other priorities.

The same can be done for the backreffed field on the `AddonDataverseNodeSettings` model as well.